app/views/search/index.html.erb

```diff
@@ -20,7 +20,7 @@
 <%= form_tag(search_path, method: :get) do %>
     <div class="input-group mb-3">
         <%= text_field_tag :search, params[:search], class: "form-control" %>
-        <button type="submit" class="btn btn-outline-secondary">
+        <button type="submit" id="submit" class="btn btn-outline-secondary">
```

Comment:
Why does the button need the id?

Reply:
An id is generally a good idea in HTML imo. In this case, we need to use it in the tests. Our linter throws an error when we use `find('button[type="submit"]).click`, we have to use `click_button` or `click_on`, which works with ids only.